google-cloud-contrib/google-cloud-nio-examples/pom.xml

```diff
@@ -12,7 +12,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-contrib</artifactId>
-    <version>0.35.1-alpha-SNAPSHOT</version>
+    <version>0.35.1-alpha-SNAPSHOT</version><!-- {x-version-update:google-cloud-pom:current} -->
```

Comment:
Should it be `google-cloud-contrib:current`?

Reply:
good catch, will fix